pallet/src/benchmarking.rs

```diff
@@ -2,144 +2,72 @@
 
 use frame_benchmarking::v2::*;
 use frame_system::{Config as SysConfig, RawOrigin};
-use pallet_balances::{Config as BalancesConfig, Pallet as Balances};
 use sp_core::crypto::Ss58Codec;
 
 use crate::{mock_utils as utils, *};
 
-const LIMIT: u128 = 60_000_000_000_000;
-
-const MAX_GAS_AMOUNT: u32 = u32::MAX;
-
 type SourceOf<T> = <<T as SysConfig>::Lookup as sp_runtime::traits::StaticLookup>::Source;
 
+macro_rules! impl_gas_costs_cal_fns {
+    ($name:tt) => {
+        pub fn $name() -> &'static [u8] {
+            core::include_bytes!(concat!(
+                "assets/move-projects/gas-costs/build/gas-costs/script_transactions/",
+                stringify!($name),
+                ".mvt"
+            ))
+        }
+    };
+}
+
 #[benchmarks(
     where
-        T: Config + SysConfig + BalancesConfig,
+        T: Config + SysConfig,
         T::AccountId: Ss58Codec,
-        T::Balance: From<u128>,
         SourceOf<T>: From<T::AccountId>,
 )]
 mod benchmarks {
     use super::*;
 
     #[benchmark]
-    fn execute(n: Linear<0, 7>) {
-        let bob_32 = utils::account::<T>(utils::BOB_ADDR);
+    fn execute(n: Linear<0, 24>) {
```

Comment:
Maybe we could leave a comment about what we did here and how we need to manually divide the gas cost by 20 in the weight.rs output file after generating the file?

Reply:
Done